lib/online_order.rb

```diff
@@ -0,0 +1,128 @@
+require_relative './order.rb'
+
+module Grocery
+
+  class OnlineOrder < Order
+    attr_reader :customer_id, :status
+
+    def initialize (id, products, customer_id, status)
+      super(id, products)
+      # @online_total = online_total
+      @status = status
+      @customer_id = customer_id
+    end
+
+    def total
+      return 10 + super
+    end
+
+    def self.find(id)
```

Comment:
Since `self.find` really doesn't change between this class and the superclass `Order` you could just not override the `find` method and use `Order`'s version.